packages/vm/lib/index.ts

```diff
@@ -212,12 +211,7 @@ export default class VM extends AsyncEventEmitter {
         Object.keys(precompiles)
           .map((k: string): Buffer => Buffer.from(k, 'hex'))
           .map((address: Buffer) =>
-            this.stateManager.putAccount(
-              address,
-              new Account({
-                balance: '0x01',
-              }),
-            ),
+            this.stateManager.putAccount(address, new Account(new BN(0), new BN(1))),
```

Comment:
I would suggest to generally avoid to use the main constructor. This makes the code less readable since one can't distinguish on the type of values directly.

Reply:
👍 done in https://github.com/ethereumjs/ethereumjs-vm/pull/911/commits/588912a2b1a5bf8b88f07613d7fec9338ac6ff51